packages/contractkit/src/wrappers/DowntimeSlasher.ts

```diff
@@ -0,0 +1,83 @@
+import { Address } from '../base'
+import { DowntimeSlasher } from '../generated/types/DowntimeSlasher'
+import {
+  BaseWrapper,
+  CeloTransactionObject,
+  proxyCall,
+  toTransactionObject,
+  valueToInt,
+} from './BaseWrapper'
+
+/**
+ * Contract handling slashing for Validator downtime
+ */
+export class DowntimeSlasherWrapper extends BaseWrapper<DowntimeSlasher> {
+  getEpochNumberOfBlock = proxyCall(
```

Comment:
we shouldn't need this. And it's duplicated in the other wrapper

Reply:
Removed.